kim_convergence/timeseries/utils.py

```
@@ -275,52 +281,56 @@ def time_series_data_uncorrelated_samples(
-                minimum_correlation_time=minimum_correlation_time)
-        except CRError:
-            raise CRError(
-                'Failed to compute the indices of uncorrelated samples of'
-                'the time_series_data.'
+                minimum_correlation_time=minimum_correlation_time,
             )
+        except CRError as e:
+            raise CRError(
+                "Failed to compute the indices of uncorrelated samples of"
+                "the time_series_data."
+            ) from e
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)